api/resources/messages.py

```diff
@@ -40,6 +56,49 @@ def get(self, *args, **kwargs):
             'data': results
         }, 200
 
+    def post(self, *args, **kwargs):
+        message, errors = self._create_message(json.loads(request.data))
+
+        if message is not None:
+            message_payload = _message_payload(message)
+            message_payload['success'] = True
+            return {
+                'success': True,
+                'data': message_payload
+            }, 201
+        else:
+            return {
+                'success': False,
+                'error': 400,
+                'errors': errors
+            }, 400
+
+    def _create_message(self, data):
+        proceed = True
+        errors = []
+
+        # employer info can't be blank
+        proceed, employer_name, errors = _validate_field(
+            data, 'employer_name', proceed, errors)
+        proceed, employer_email, errors = _validate_field(
+            data, 'employer_email', proceed, errors)
+        proceed, applicant_id, errors = _validate_field(
+            data, 'applicant_id', proceed, errors)
```

Comment:
I tried seeing what the errors look like locally via postman, but the only descriptive error was when I left `applicant_id` blank: `required 'applicant_id' parameter is missing`.
when leaving `body` blank, it renders an `internal server error`.
otherwise, the post request successfully went through when I left `employer_name` and `employer_email` blank.